reset_password.go

```diff
@@ -39,6 +39,7 @@ type Credentials struct {
 	Password string `json:"passcode"`
 }
 
```

Comment:
Some of the methods below could use better comments, if you have time. :)

Reply:
lol, the methods names are self documenting but to write idiomatic/lint-able go code where all the public methods and functions are documented the comment tends to be redundant.